### PR TITLE
Modify animal table

### PR DESCRIPTION
**Pull request by GordonOtieno**
### Added query and update animals table
- Added species column.
- Inserted records into the database.
-  Updated records in the database.
- Used Postgresql aggregation methods
- Added screenshots: https://drive.google.com/drive/folders/1lJE7DyCufsqTr4YuvegYsQVRwCXxk-gG

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,11 +1,53 @@
...
+UPDATE animals SET weight_kg = (weight_kg*-1) WHERE weight_kg <0;
+COMMIT;
+-- take screenshort 3
+
+-- tasks3 query 1
+SELECT COUNT(*) AS number_of_animals FROM animals;
+SELECT COUNT(*) AS never_tried_escape FROM animals WHERE escape_attempts=0;
+SELECT avg(weight_kg) AS average_weight from animals;
+-- Who escapes the most, neutered or not neutered animals?
+SELECT neutered, COUNT(escape_attempts) FROM animals GROUP BY neutered;
```
Comment (sja-thedude) [inline queries.sql]: - [ ] In this query, instead of COUNT you should use MAX or AVG as we have to find who escapes the most.

File: queries.sql [hunk 2]
```diff
@@ -1,11 +1,53 @@
...
+-- What is the minimum and maximum weight of each type of animal?
+SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
+
+-- What is the average number of escape attempts per animal type of those born between 1990 and 2000?
+SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-01-01' GROUP BY species;
```
Comment (sja-thedude) [inline queries.sql]: - [ ] In this query, the end date should be 31 Dec 2000 as we have to find the average number of escape attempts born between 1990 and 2000 so it should include the entire year 2000.

File: queries.sql [hunk 3]
```diff
@@ -1,11 +1,53 @@
...
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+SELECT * FROM ANIMALS;
+-- take screenshort 1
+
+-- tasks2 query 3
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
```
Comment (sja-thedude) [inline queries.sql:31]: - [ ] You have to add this query `SELECT * FROM ANIMALS;` before rollback also as after deleting we have to check whether the table is deleted and then after rollback we will check if its again restored.

File: queries.sql [hunk 4]
```diff
@@ -1,11 +1,53 @@
...
+SELECT * FROM ANIMALS;
+ROLLBACK;
+
+-- tasks2 query 2
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+SELECT * FROM ANIMALS;
+-- take screenshort 1
```
Comment (sja-thedude) [inline queries.sql]: - [ ] For both these queries you have to verify that the change is made before rollback and verify that the change is undone after rollback. `SELECT species from animals;`
Basically you have to add this before and after rollback.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @GordonOtieno, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional Readme
✔️ Correct Git flow

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @GordonOtieno, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**